### DISCUSSION TITLE
Cant Start LinuxGSM[BUG]

### DISCUSSION THREAD

**Discussion by amimbitassu** [GameServerManagers/LinuxGSM#3646] [General]
## User Story

Trying to run Mordhau Server.

## Basic info

* **Distro:** Ubuntu 18.04.06 & Ubuntu 20.04 & Centos 8
* **Game:** Mordhau
* **Command:** Start
* **LinuxGSM version:** v21.4.0

## Further Information

Tried fresh install of Ubuntu&LinuxGSM. (Both 21.04 and 18.04)
The server seems like it starts on first ./mhserver start command.(I'm not exacly sure it actually starts, it says so but I did not try to connect to the server.) When console or any other command is run, the server stops. Can not run unless server is rebooted.

Script Logs:
Oct 27 07:46:28.092 mhserver: START: FATAL: Unable to start LinuxGSM
Oct 27 07:46:28.111 mhserver: START: INFO: LinuxGSM version: v21.4.0
Oct 27 07:46:28.115 mhserver: START: FATAL: core_exit.sh exiting with code: 1

Initial start log where it seems like server started.  When I run the console command it says server is not running.

Oct 27 06:58:46.567 mhserver: START: PASS: Started LinuxGSM
Oct 27 06:58:46.592 mhserver: START: INFO: LinuxGSM version: v21.4.0
Oct 27 06:58:46.596 mhserver: START: PASS: core_exit.sh exiting with code: 0
Oct 27 06:59:01.800 mhserver: CORE: PASS: Downloading command_console.sh
Oct 27 06:59:06.081 mhserver: CONSOLE: ERROR: Failed to access: Server not runn$
Oct 27 06:59:10.891 mhserver: START: INFO: Using anonymous Steam login
Oct 27 06:59:11.706 mhserver: START: INFO: Rotating log files

There are 6 console .log's all of which are empty inside.
There are no .log files inside server folder.
I've recreated same results with Centos 8 as well.

**This is how debug starts if it helps;**

mhserver@mordhauserver:~$ ./mhserver debug

MORDHAU Debuging
=================================
Distro:       Ubuntu 18.04.6 LTS
Arch:         x86_64
Kernel:       4.15.0-161-generic
Hostname:     mordhauserver
tmux:         tmux 2.6
Avg Load:     0.11, 0.24, 0.14
Free Memory:  7.4GB
Free Disk:    35G
glibc required: 2.17
Game Server IP: 0.0.0.0:7777
Internet IP:    *.*.*.*:7777 (Original IP was here, I masked it.)
Server password:        NOT SET
Start parameters:
 ./MordhauServer-Linux-Shipping Mordhau FFA_ThePit -log -MultiHome=0.0.0.0 -Port=7777 -BeaconPort=15000 -QueryPort=27015 -GameIni=/home/mhserver/serverfiles/Mordhau/Saved/Config/LinuxServer/Game.ini

Use for identifying server issues only!
Press CTRL+c to drop out of debug mode.
Warning! If mhserver is already running it will be stopped.

Continue? [Y/n] Y

**After I pres enter to approve;**

[  OK  ] Debuging mhserver: Stopping debug

**Debugging is stopped.**

~~I've come across some threads about tmux and double quotes. Could it be a similar problem with my setup? Maybe due to localisation or input language of the server? How can i identify this?~~
There are two tmux logs in the main folder. Maybe there are important information here?
tmux client log : https://termbin.com/e20u
tmux server log : https://termbin.com/erbm

I think i was able to create a debug log with developer mode : https://termbin.com/vs22

I dont know where else to check to understand what is causing error.

**Comment by Claiyc** [comment]
Can you provide any console or server logs?

This may be moved to a discussion as it's most likely not an issue with lgsm.

**Reply by johnoclockdk** [reply]
it was moved

**Comment by amimbitassu** [comment]
Sharing updates,
I've tried exacly same procedure with digitalocean 1core 1gb ram 25gb ssd vm. Ubuntu 18.04. No problem at all. Can run as soon as the ./mhserver install script is finished.
At a fresh ubuntu install, used digitalocean.ubuntu repositories, no luck.
Tried changing systemtime zone, no luck.
I think something with localisation is causing this problem?

**Comment by amimbitassu** [comment]
In the tmux client logs it says connection refused multiple times? Maybe thats the cause?

**Comment by skylerspaeth** [comment]
@amimbitassu did you end up figuring out conclusively what caused this? I'm having the same problem trying to setup CS:GO server, even with loads of resources allocated.
